utils/handlers/config_eventing.go

```diff
@@ -0,0 +1,152 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/spaceuptech/space-cloud/config"
+	"github.com/spaceuptech/space-cloud/utils/admin"
+	"github.com/spaceuptech/space-cloud/utils/syncman"
+)
+
+// HandleAddService is an endpoint handler which deletes a table in specified database
+func HandleAddRule(adminMan *admin.Manager, syncMan *syncman.Manager) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		// Get the JWT token from header
+		tokens, ok := r.Header["Authorization"]
+		if !ok {
+			tokens = []string{""}
+		}
+		token := strings.TrimPrefix(tokens[0], "Bearer ")
+
+		// Check if the request is authorised
+		if err := adminMan.IsTokenValid(token); err != nil {
+			w.WriteHeader(http.StatusUnauthorized)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		vars := mux.Vars(r)
+		name := vars["name"]
+		project := vars["project"]
+
+		conf, err := syncMan.GetConfig(project)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		value := &config.EventingRule{}
+		json.NewDecoder(r.Body).Decode(&value)
+
+		conf.Modules.Eventing.Rules[name] = *value
+
+		syncMan.SetProjectConfig(conf)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			json.NewEncoder(w).Encode(map[string]string{"error": err.Error()})
+			return
+		}
+
+		w.WriteHeader(http.StatusOK) //http status codee
+		json.NewEncoder(w).Encode(map[string]interface{}{})
+		return
+
+	}
+}
+
+// HandleDeleteService is an endpoint handler which deletes a table in specified database
+func HandleDeleteRule(adminMan *admin.Manager, syncMan *syncman.Manager) http.HandlerFunc {
```

Comment:
```suggestion
func HandleDeleteEventingRule(adminMan *admin.Manager, syncMan *syncman.Manager) http.HandlerFunc {
```
and change the comment to describe the function